apps/s_server.c

```diff
@@ -961,6 +970,11 @@ const OPTIONS s_server_options[] = {
     {"ktls", OPT_KTLS, '-', "Enable Kernel TLS for sending and receiving"},
     {"sendfile", OPT_SENDFILE, '-', "Use sendfile to response file with -WWW"},
 #endif
+#ifndef OPENSSL_NO_RPK
+    {"enable_server_rpk", OPT_ENABLE_SERVER_RPK, '-', "Enable raw public keys (RFC7250) from the server"},
+    {"enable_client_rpk", OPT_ENABLE_CLIENT_RPK, '-', "Enable raw public keys (RFC7250) from the client"},
+    {"peer_rpk", OPT_PEER_RPK, '<', "PEM-encoded certificate with server RPK"},
```

Comment:
This is the client's RPK, not the server's.
(And it looks like it could be "native" in the file, not just as a certificate, too.)